internal/venice-client-common/src/main/java/com/linkedin/venice/utils/AvroSchemaUtils.java

```diff
@@ -365,4 +368,18 @@ private static boolean hasDocFieldChangeFields(Schema s1, Schema s2) {
     }
     return false;
   }
+
+  public static void validateTopLevelFieldDefaultsValueRecordSchema(Schema valueRecordSchema) {
+    Validate.notNull(valueRecordSchema);
+    if (valueRecordSchema.getType() != RECORD) {
+      return;
+    }
+    for (Schema.Field field: valueRecordSchema.getFields()) {
+      if (!AvroCompatibilityHelper.fieldHasDefault(field)) {
+        throw new IllegalArgumentException(
+            "Schema must have default in each top-level field. schema: " + valueRecordSchema + " field: "
```

Comment:
If we want to put schema content in the exception, can we put `field.name()` before printing out the schema content, so that the operator will know the problematic field without scrolling down to the end of the schema?

Reply:
Do you consider this small tweak?
Many time in prod, it is very tricky to skip the full content of schema, and then find out the problematic field since the schema content could be fairly large: several KB or even 100+KB.

Reply:
I did, forgot to add the change, either ways this is not called now as I removed the check in addValueSchema